### ISSUE TITLE
[BUG]Multiple side-mission not counted correctly in master_daily branch

### ISSUE THREAD

**Issue by GoldJohnKing** [Vdauphin/HeartsAndMinds#755]
**Arma 3 Version:** `1.94` (stable)
**CBA Version:** `3.12.0` (stable)
**ACE3 Version:** `3.12.6` (stable)
**RHS USAF Version:** `0.4.9` (stable)
**Hearts and minds Version:** `master_daily branch` (commit e6832f8)

**Mods:**
```
- CBA_A3
- ace
- RHS USAF
```

**Description:**
- As you suggest, I've switched to master_daily branch to test new functions.
- I found multiple side-mission feature might have counting issue.
- I set btc_p_side_mission_cycle to 3 in param.hpp, but finally 5 missions was generated (I've excluded those sub-missions). When I set it to 2, finally 3 missions were generated.
- The actually generated side mission count always equals to 2 * btc_p_side_mission_cycle - 1.
- I've used the delete database function and restarted the mission for several times, it still happens.

**Steps to reproduce:**
- Set btc_p_side_mission_cycle greater than 1 in param.hpp.
- Start the mission, you'll find actually generated side-mission count always equals to 2 * btc_p_side_mission_cycle - 1.

**Comment by Vdauphin** [comment]
> Set btc_p_side_mission_cycle greater than 1 in param.hpp.

I set 3 side mission at the same time and I have 3 side mission at the same time:

![20190815185418_1](https://user-images.githubusercontent.com/14364400/63111546-38294200-bf8e-11e9-89dd-95f58acd44de.jpg)
![20190815185308_1](https://user-images.githubusercontent.com/14364400/63111564-3f505000-bf8e-11e9-96a1-9c0e6022212e.jpg)

**Comment by GoldJohnKing** [comment]
That's odd. We use 1 or 2 headless client on our server. Could it be related to headless client?

**Comment by Vdauphin** [comment]
No

**Comment by GoldJohnKing** [comment]
I have this problem all the time...
![107410_20190816011738_1](https://user-images.githubusercontent.com/26923626/63112910-fa471080-bfc3-11e9-9b41-7aee9bb2fef9.png)

**Comment by Vdauphin** [comment]
try to reproduce the issue with using only `@CBA_A3`, `@ace`  on the master_daily and unchanged branch (only necessary parameter)

**Comment by Vdauphin** [comment]
With HC and set with 3 side mission cycle:
![20190815193921_1](https://user-images.githubusercontent.com/14364400/63114192-817c9000-bf94-11e9-91de-36c95221ef7a.jpg)
![20190815193932_1](https://user-images.githubusercontent.com/14364400/63114193-82152680-bf94-11e9-95a7-2124a28cc3ae.jpg)

**Comment by GoldJohnKing** [comment]
I've found out what caused the problem... 😅 

If I use “Land_RepairDepot_01_tan_F” or similar objects replace the "btc_create_object" object, mission counts would be double -1, the ACE interaction options at "btc_create_object" would be double, too.

This one:
![107410_20190816130154_1](https://user-images.githubusercontent.com/26923626/63144694-a6214800-c026-11e9-8504-73e3a6a4a140.png)

I've tried using larger objects like carrier or submarine and other stuffs in Tanks DLC, but neither of them double the counts. Only “Land_RepairDepot_01_tan_F” would double the side-mission counts and ACE interaction counts.

Wired... I just wonder why...

**Comment by Vdauphin** [comment]
> If I use “Land_RepairDepot_01_tan_F” or similar objects replace the "btc_create_object" object, mission counts would be double -1

Side mission system isn't link to btc_create_object

> the ACE interaction options at "btc_create_object" would be double, too.

Why?

> Only “Land_RepairDepot_01_tan_F” would double the side-mission counts and ACE interaction counts.

I just replace to the “Land_RepairDepot_01_tan_F” and the number of side mission doesn't change. You are still doing something else which is generating a bug in your branch

Close?

**Comment by GoldJohnKing** [comment]
I'm pretty sure it's related to “Land_RepairDepot_01_tan_F”.

I used unchanged Altis branch, only replaced btc_create_object to “Land_RepairDepot_01_tan_F”.

The first time I join the mission, side-mission count is correct.

Then disconnect, restart the server, and join again. Side-mission count would be double minus 1.

Then this would happen all the time.

I also enabled save and load parameters. Saving and loading functions together with btc_create_object replaced by “Land_RepairDepot_01_tan_F” might caused the problem.

I think close this issue would be better for now, until I found more detailed information about it.
